src/kokkos_ext/ArborX_DetailsKokkosExtSort.hpp

```diff
@@ -78,6 +79,8 @@ namespace KokkosExt
 template <typename ExecutionSpace, typename Keys, typename Values>
 void sortByKey(ExecutionSpace const &space, Keys &keys, Values &values)
 {
+  KokkosExt::ScopedProfileRegion guard("ArborX::KokkosExt::sortByKey::Kokkos");
```

Comment:
Would say "KokkosBinSort" but not blocking.

Reply:
I thought about it, but thought we may switch to using Kokkos Thrust at some point in the future, so decided to leave it as generic Kokkos.